src/middlewares/auth.middleware.js

```diff
@@ -11,6 +11,14 @@ const authMiddleware = {
     }
     next();
   },
+  doesHaveManager: async (req, res, next) => {
```

Comment:
`doesHaveManager` is not a good function name. Ensure your function name has a verb and clearly describes what the function does.